src/Pos/Communication/Limits.hs

```diff
@@ -283,6 +284,12 @@ instance MessageLimited (MsgHeaders ssc) where
         return $
             MsgHeaders <$> vectorOf Const.recoveryHeadersMessage headerLimit
 
+-- TODO: Update once we move to CBOR.
+instance MessageLimitedPure MsgSubscribe where
+    msgLenLimit = 0
```

Comment:
@dcoutts Interestingly enough, it does not (I wrote this code, but I have simply converted it from what the `store` implementation was doing, so bear with me):

https://github.com/input-output-hk/cardano-sl/blob/master/core/Pos/Binary/Class/Core.hs#L167-L169